.github/workflows/Labeler.yml

```diff
@@ -5,7 +5,8 @@ on:
     branches: [ master, stable ]
 
 # Declare default permissions as none.
-permissions: none
+permissions:
+  contents: read
```

Comment:
This should resolve #39.

Reply:
❗ https://github.com/reactive-firewall/python-repo/actions/runs/9913946428/workflow#L8 - This is an invalid config apparently. Will need to promote the default permissions to be read-all 👎🏻
🤔 This is unfortunate, but acceptable improvement over read-write for the open-source repo. 🤷🏻